app/models/forecast_snapshot.rb

```diff
@@ -114,8 +115,10 @@ def delete_file_from_disk(name)
     begin
       File.delete(path(name))
     rescue StandardError => e
-      Rails.logger.error e.message
-      return false
+      if e.message !~ /no such file/i
+        Rails.logger.error e.message
+        return false
```

Comment:
We should probably still log the no such file. I believe that's an error case as well.